### PR TITLE
task #6 - prepare code to do something nice

### PR DESCRIPTION
**Pull request by NormalDeveloperNumber1**
Please chack it (CR) and merge or back to corrections :)

### CODE REVIEW

File: kuku.py [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
-print "End Of Code ..."
-
-print "KOMIT 1 - branch testi444ng"
-print "KOMIT 2 - branch testing444"
-print "KOMIT 3 - branch testin55g"
-
-print "Additioanl line in the end of file"
-print "and here"
+print
+print
```
Comment (HighLifeComboDeveloper) [inline kuku.py]: Why so much prints ?

File: kuku.py [hunk 2]
```diff
@@ -1,22 +1,11 @@
...
-
-print "Additioanl line in the end of file"
-print "and here"
+print
+print 
+print "-------------------------------------------------------"
+print "Here we can add some technical system information later"
+print "-------------------------------------------------------"
+print
+#for i in range (1,11):
```
Comment (HighLifeComboDeveloper) [inline kuku.py]: Why did you leave this loop in the comment ?

File: kuku.py [hunk 3]
```diff
@@ -1,22 +1,11 @@
...
+print
+print 
+print "-------------------------------------------------------"
+print "Here we can add some technical system information later"
+print "-------------------------------------------------------"
+print
+#for i in range (1,11): 
+#	print "Loop #KUKU",i
+#	print "******"
+#print
```
Comment (HighLifeComboDeveloper) [inline kuku.py]: print as a comment in the last line ?

File: kuku.py [hunk 4]
```diff
@@ -1,22 +1,11 @@
 #!/usr/bin/python
-print "USR2 dodaje koment do mastera"
-print "USR1 is commited to master"
-print 
-print "It will be not the best software ...  ...."
```
Comment (HighLifeComboDeveloper) [inline kuku.py]: It was really nice line I thin . don;t you ?

Comment (NormalDeveloperNumber1) [reply kuku.py]: Yes, I will leave it in new code...

File: kuku.py (hunk 1 above)
Comment (NormalDeveloperNumber1) [reply kuku.py]: I think data presentation will be better on the screen

File: kuku.py (hunk 2 above)
Comment (NormalDeveloperNumber1) [reply kuku.py]: It's to not forgotten how to build loops in the python :)

File: kuku.py (hunk 3 above)
Comment (NormalDeveloperNumber1) [reply kuku.py]: i't was stupid. iw ill correct it...

**Comment by HighLifeComboDeveloper** [comment]
It's cocreected. Could you check it again plaese ?

File: kuku.py [hunk 5]
```diff
@@ -8,4 +9,3 @@
 #for i in range (1,11):
```
Comment (HighLifeComboDeveloper) [inline kuku.py]: Seems ok now ... go ahead

**Pull request merged**